test/bdd/pkg/edv/edv_steps.go

```diff
@@ -36,6 +37,8 @@ const (
 
 	documentTypeExpectedStructuredDoc  = "expected Structured Document"
 	documentTypeDecryptedStructuredDoc = "decrypted Structured Document"
+
+	testReferenceID = "urn:uuid:abc5a436-21f9-4b4c-857d-1f5569b2600d"
```

Comment:
Similar comment as earlier regarding making the referenceID something other than a UUID

Reply:
Also note that the reference ID is optional... you could leave it out if there's nothing to really demonstrate with it here.

Reply:
fixed